tbx/static_src/sass/base/_typography.scss

```diff
@@ -42,14 +42,17 @@ h6 {
     @include heading-text();
     font-weight: $weight--light;
 
+    // Legacy CMS used italics in headings so reset that here
+    i {
+        font-style: normal;
```

Comment:
Should we also reset the bold here?

Reply:
I can add that in that will take effect in heading blocks. We'll need to set it separately for rich text headings as they don't get the `.heading` class applied

Reply:
Added in https://github.com/torchbox/torchbox.com/pull/80/commits/3c3f4a73f109af533e78eac65820d79aee268c44